src/Shared/Modules/BlazorBoilerplate.Theme.Material.Admin/Pages/Admin/IdentityResources.razor

```diff
@@ -12,15 +12,15 @@
 <h1>@L["IdentityResources"]</h1>
 <p>Identity Resources Management.</p>
 
-@if (identiyResources == null)
+@if (identityResources == null)
```

Comment:
This file should not be included in PR.

Reply:
Agree that was a separate fix I did to rename misspelled idenity to identity. I will send in separate PR.